totest-manager.py

```diff
@@ -920,11 +920,11 @@ def __init__(self, *args, **kwargs):
         }
         self.openqa_server = {
             'openSUSE': 'https://openqa.opensuse.org',
-            'SLE': 'https://openqa.suse.de',
+            'SUSE': 'https://openqa.suse.de',
```

Comment:
Can you explain why this can be done now?

Reply:
For some reason you introduced the args `SLE` and `openSUSE` for the `--project-base` argument and defaulted to `openSUSE`.
`SLE` doesn't fit nicely, because the IBS project actually starts with `SUSE:`

Reply:
ok, don't know anymore why I did that, fine ;)